riva/utils/wav/wav_reader.cc

```diff
@@ -105,6 +109,14 @@ ParseHeader(
     encoding = nr::FLAC;
     samplerate = 16000;
     channels = 1;
+    data_offset = file_stream.tellg();
+    return true;
+  } else if (header.file_tag == "OggS") {
+    // TODO parse sample rate and channels from stream
+    encoding = nr::OGGOPUS;
+    samplerate = 16000;
```

Comment:
should these values be parsed from the header?

Reply:
fixed